lib/item_stream_handler.rb

```diff
@@ -61,9 +62,10 @@ def handle (event)
     update_tally_if_necessary
     checkout_count = 0
 
-    event["Records"]
+    records = event["Records"]
       .select { |record| record["eventSource"] == "aws:kinesis" }
-      .each do |record|
+    records = PreProcessingRandomizationUtil.send(ENV['RANDOMIZATION_METHOD'], records)
```

Comment:
Does this mean you send the second parameter (here it is `records`) to the first parameter (here it is `ENV['RANDOMIZATION_METHOD']`) ? You send the method to the class `PreProcessingRandomizationUtil` but I don't see any method in `PreProcessingRandomizationUtil` will take the value of `ENV['RANDOMIZATION_METHOD']`